Demo/Application/Features/Venmo - Custom Button/BraintreeDemoCustomVenmoButtonViewController.m

```diff
@@ -14,7 +14,6 @@ - (void)viewDidLoad {
     [super viewDidLoad];
     self.venmoDriver = [[BTVenmoDriver alloc] initWithAPIClient:self.apiClient];
     self.title = NSLocalizedString(@"Custom Venmo Button", nil);
-    self.paymentButton.hidden = ![self.venmoDriver isiOSAppAvailableForAppSwitch];
```

Comment:
In order to maintain a UI test that checks for a message saying that the Venmo app is not installed, we needed to make the Venmo button always visible.